runtime/src/snapshot_utils.rs

```diff
@@ -999,8 +1000,9 @@ pub fn process_accounts_package_pre(
 
     let hash = accounts_package.hash; // temporarily remaining here
     if let Some(expected_hash) = accounts_package.hash_for_testing {
+        let sorted_storages = SortedStorages::new(&accounts_package.storages);
```

Comment:
Is `process_accounts_package_pre()` only called when a testing CLI option is enabled too? Asking because this fn calls regular `SortedStorages::new()`, which has this comment:

https://github.com/solana-labs/solana/blob/cf20c7990d8fdbbf7dcf2d0ebb09e5e23ad1bf00/runtime/src/sorted_storages.rs#L35-L39

Reply:
No - the function is now being used in a place where the comment doesn't apply. I need to fix that.